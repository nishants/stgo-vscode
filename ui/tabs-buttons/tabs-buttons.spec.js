import React from 'react';
import {shallow} from 'enzyme';

import TabButtons from './index';

describe('ui/tabs-buttons', () => {
  it('should render tab buttons', () => {
    const wrapped = shallow(<TabButtons/>);
    expect(wrapped).toMatchSnapshot();
  });
});