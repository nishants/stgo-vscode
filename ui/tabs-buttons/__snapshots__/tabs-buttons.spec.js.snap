// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/tabs-buttons should render tab buttons 1`] = `
<div
  className="tab-buttons"
>
  <button
    className="tab"
    key="overview"
    onClick={[Function]}
  >
    Overview
  </button>
  <button
    className="tab"
    key="cypressCi"
    onClick={[Function]}
  >
    Cypress CI
  </button>
  <button
    className="tab"
    key="screenshotDiffs"
    onClick={[Function]}
  >
    Screenshot Differences
  </button>
  <button
    className="tab"
    key="ciLogs"
    onClick={[Function]}
  >
    Branch Logs
  </button>
</div>
`;
